pages/settings.py

```diff
@@ -0,0 +1,193 @@
+
+            def add_skill(self, skill_name):
+                element = self._root_element.find_element(*self._skills_field_locator)
+                element.send_keys(skill_name)
+                element.send_keys(Keys.RETURN)
+
+            @property
+            def delete_skill_buttons(self):
+                return self._root_element.find_elements(*self._delete_skill_buttons_locator)
+
+            def click_update(self):
+                self._root_element.find_element(*self._update_locator).click()
+
+    class YouAndMozilla(PageRegion):
+
+        _contributions_form_locator = (By.CSS_SELECTOR, "form.edit-profile:nth-child(1)")
+
+        @property
+        def contributions(self):
+            return self.Contributions(self.base_url, self.selenium,
+                                        self._root_element.find_element(*self._contributions_form_locator))
+
+        class Contributions(PageRegion):
+
+            _select_month_locator = (By.ID, 'id_date_mozillian_month')
+            _select_year_locator = (By.ID, 'id_date_mozillian_year')
+            _month_locator = (By.CSS_SELECTOR, '#id_date_mozillian_month > option')
+            _year_locator = (By.CSS_SELECTOR, '#id_date_mozillian_year > option')
+            _selected_month_locator = (By.CSS_SELECTOR, '#id_date_mozillian_month > option[selected="selected"]')
+            _selected_year_locator = (By.CSS_SELECTOR, '#id_date_mozillian_year > option[selected="selected"]')
+            _update_locator = (By.ID, 'form-submit-top')
+
+            def select_month(self, option_month):
+                element = self._root_element.find_element(*self._select_month_locator)
+                select = Select(element)
+                select.select_by_value(option_month)
+        
+            def select_year(self, option_year):
+                element = self._root_element.find_element(*self._select_year_locator)
+                select = Select(element)
+                select.select_by_value(option_year)
+        
+            @property
+            def month(self):
+                return self._root_element.find_element(*self._selected_month_locator).text
+        
+            @property
+            def year(self):
+                return self._root_element.find_element(*self._selected_year_locator).text
+        
+            @property
+            def months_values(self):
+                return [month.get_attribute('value') for month in self._root_element.find_elements(*self._month_locator)]
+        
+            def select_random_month(self):
+                return self.select_month(random.choice(self.months_values[1:]))
+
+            def select_random_year(self):
+                return self.select_year(random.choice(self.years_values[1:]))
+
+            @property
+            def years_values(self):
+                return [year.get_attribute('value') for year in self._root_element.find_elements(*self._year_locator)]
```

Comment:
It's not clear the difference between `def year` and `def years_values`. At minimum, add code comments clarifying their usage.